stages/org.osbuild.modprobe

```diff
@@ -25,69 +20,61 @@ import osbuild.api
 
 SCHEMA = r"""
 "additionalProperties": false,
+"required": ["commands","filename"],
 "properties": {
-  "configuration_files": {
+  "filename": {
+    "type": "string",
+    "description": "Name of the modprobe configuration file.",
+    "pattern": "^[\\w.-]{1,250}\\.conf$"
+  },
+  "commands": {
+    "required": ["commands"],
```

Comment:
This `required` property is most probably a bug, because there is no property with such name in this object.